### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by i8wu**
Performance is not very good with getLayout(), using getTranslateTransform seems to drastically improve performance. Also added useNativeDriver for opacity. Please let me know if you want anything else changed before this can be merged! Addresses #33

### CODE REVIEW

**Comment by gitim** [comment]
Thanks, for your pr! 

> Performance is not very good with getLayout(), using getTranslateTransform seems to drastically improve performance.

hmm, interesting, I will check out

File: src/Row.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default class Row extends Component {
   _relocate(nextLocation, animated) {
     this._location = nextLocation;
 
-    if (animated) {
+    if (animated && !this._isAnimationRunning) {
```
Comment (gitim) [inline src/Row.js]: this is not correct, there is possible case, when location set programmatically, before previous relocation animation not ended. Also I doubt, that this change hugely improve performance

File: src/Row.js [hunk 2]
```diff
@@ -156,7 +156,7 @@ export default class Row extends Component {
     return (
       <Animated.View
         {...this._panResponder.panHandlers}
-        style={[style, styles.container, this._animatedLocation.getLayout()]}
+        style={[style, styles.container, {transform: this._animatedLocation.getTranslateTransform()}]}
```
Comment (gitim) [inline src/Row.js]: I didn't notice any change in animation performance. why it has to improve it, is it mentioned anywhere? I know that in browsers transforms give much smoother animation than positioning with top/left, but as for as I know it is not true for react native

Comment (i8wu) [inline src/Row.js]: What are you testing with? Testing on an actual Android device shows significant improvements for me. I can't find it documented anywhere why it's better, but it seems to be the case. It's not very noticeably better on iOS though, but Animated has been better for iOS in my experience.

File: src/Row.js (hunk 1 above)
Comment (i8wu) [inline src/Row.js]: Didn't realize this change would miss that case, I can remove it.

File: src/Row.js (hunk 2 above)
Comment (ItsNoHax) [inline src/Row.js]: Gonna try his branch out and report how it felt it felt for me.

Comment (ItsNoHax) [inline src/Row.js]: I didn't notice any significant improvement using a Samsung Galaxy S7, how big are your lists?

Comment (i8wu) [inline src/Row.js]: less than 10 items

Comment (i8wu) [inline src/Row.js]: Actually it seems like rnrf allows my other scene's render to still be called while I'm in the ListView scene. After minimizing the render calls the performance for sortable-list is better. Maybe this fix is only useful under more strenuous conditions?

**Comment by gitim** [comment]
I don't see any solid reason why to start use transform, `useNativeDriver: true` is not required any more, because we removed opacity animation, stop autoScroll if scroll disabled fix already merged. So, close this pr.

**Pull request closed without merging**